.github/workflows/windows.yml

```diff
@@ -0,0 +1,184 @@
+name: Windows
+
+on: [push, pull_request]
+
+jobs:
+  msvc:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        VER: [v141, v142, v143]
+        EXT: [ON] # OFF: error C2079: 'statbuf' uses undefined struct 'stat'
```

Comment:
I don't observe this error when enabling the OFF switch, just warnings.

Reply:
Then something must've changed in the CRT, because this set of macro guards on `stat` previously evaluated to false, hiding the type.
```
#if defined(_CRT_INTERNAL_NONSTDC_NAMES) && _CRT_INTERNAL_NONSTDC_NAMES && !defined _CRT_NO_TIME_T
    struct stat
    { ... };
#endif
```